src/ClusterBootstrap/services/restfulapi/config.yaml

```diff
@@ -70,14 +70,14 @@ elasticsearch:
   {% for elasticsearch_node in cnf["elasticsearch_node"] %}
   - 'http://{{elasticsearch_node}}:{{cnf["elasticsearch"]["port"]["http"]}}'
   {% endfor %}
-{% if cnf["__extract_job_log_legacy"] %}
-__extract_job_log_legacy: True
+{% if "__extract_job_log_legacy" in cnf %}
+__extract_job_log_legacy: {{ cnf["__extract_job_log_legacy"] }}
```

Comment:
can you also modify in `ClusterBootstrap/services/jobmanager/config.yaml`?

Reply:
updated